src/swap/components/SwapSettingsSlippageInput.tsx

```diff
@@ -12,10 +12,21 @@ export function SwapSettingsSlippageInput({
   className,
   defaultSlippage = 3,
```

Comment:
Something I am reflecting more and more, is where `defaultSlippage` should be set.

If at top level `Swap` component. Or in this component.

The reason is that we need a default value in the `SwapProvider`, but we actually set it here.

So here an open question, what's the developer experience we want to offer? Counting we need to set the slippage but also need to have that value as default for the input.

Reply:
`SwapSettingsSlippageInput` is an optional sub-component, therefore I think we need to set defaultSlippage at the top level of the `Swap` component

Reply:
We can move the defaultSlippage parameter to the outer layer and always set it in the top level

Reply:
Ok so are we thinking about
```
<Swap maxSlippage={} >
```

and use that for the both the default value and maxSlippage value?

Reply:
or as we will have several default options, we should consider to have a more approach on this.

And do
```
<Swap options={} >
```

and is `options` the right word, or should be `settings`?

Reply:
@abcrane123 curis to hear your thoughts here as well.

Reply:
How about: 

`<Swap defaultMaxSlippage={} >`

This would be used as the `defaultMaxSlippage` value. 

If the developer includes the `SwapSettingsSlippageInput` sub-component, then the input `maxSlippage` values will override the `defaultMaxSlippage`.

Reply:
good point 👍

Reply:
@Zizzamia I can add this in when I update the `Swap.tsx` file in my following PR

Reply:
I like `options` prop IMO

Reply:
Yeah, I am 50/50 between `options` and `settings`. If we go with `options` than we need to make sure it will be use as new best practices in other componetns as well.

So let's go with `options`.

Reply:
Btw, in the next PR, let's make sure we remove `defaultSlippage`, and get the info about `maxSlippage` from `lifeCycleStatus.statusData.maxSlippage`.

@cpcramer let me super clear, we don't want this prop, because it will confuse people.